Add some missing c23, c36 and OP translations (#96)
IllusionMods/KoikatsuTranslation: translation/en/RedirectedResources/assets/abdata/adv/scenario/c36/20/303/translation.txt

// Dumped for Koikatsu v5.1 (CharaStudio) by KK_TextDump v1.4.6.0
//
「ふぅ、大分歌ったわね。そろそろ時間かしら」="*Pant* we've sung a lot. I think the time is about up."

> CHOICE:『延長して、もうちょっと歌おうよ』=Extend it, let's sing a little more

Reviewer: This refers to "getting an extension" on the room when doing Karaoke

```suggestion
CHOICE:『延長して、もうちょっと歌おうよ』=Let's get an extension, and sing a bit longer.
```
